pkgs/applications/misc/cura/default.nix

```diff
@@ -3,20 +3,20 @@
 
 mkDerivation rec {
   pname = "cura";
-  version = "4.12.1";
+  version = "4.13";
 
   src = fetchFromGitHub {
     owner = "Ultimaker";
     repo = "Cura";
     rev = version;
-    sha256 = "sha256-QvX9o1nrYmY6zzPcxl+xD6JTMdphzT/is1SMYrISu4o=";
+    sha256 = "sha256-qopY532pNudxaqOiEA5z71TP4Ckp69fZQbK1rghS3JY=";
   };
 
   materials = fetchFromGitHub {
     owner = "Ultimaker";
     repo = "fdm_materials";
     rev = version;
-    sha256 = "0ykf14j4yx4cf12qw0d4bff9ixrx96m6wxqvi83sn721y7dsd2rs";
```

Comment:
Okay, after some experiments, I think this sha256 (`0ykf14j4yx4cf12qw0d4bff9ixrx96m6wxqvi83sn721y7dsd2rs`) is the sha256 of the [4.10.0](https://github.com/Ultimaker/fdm_materials/releases/tag/4.10.0) tarball...

Reply:
seems like i forgot to update this in my PR: #150544